_common/qiskit/benchmark_runner.py

```diff
@@ -88,6 +93,26 @@
 
     args = parser.parse_args()
 
+# For Inserting the Noise model default into exec option as it is function call 
+    if args.noise_model != None :
+        
+        if args.noise_model == 'None':
+           args.exec_options["noise_model"] = None 
+        else:
+            module,method = args.noise_model.split(".")
```

Comment:
Overall, I'm not sure I like this approach. There's a lot of fragility here that expects things to be in a format that, if any different, will completely fail. The splitting of string, use of globals, string checking for "None" over `None`, equality checks of `== None` over `is None`, etc. 

I would suggest we consider another way of supporting this option. Perhaps this would entail imposing more structure from what we expect the user to provide, or, take another approach that doesn't use `argparse` at all but simply uses a configuration file of some type. Thoughts @rtvuser1  / @nithingovindugari ?

Reply:
Hi @vprusso I have corrected your typo suggestions and regarding the None type declaring it as 'None' as in argeparse even if the input is given as None from the command line by default, it is assumed as a string. Their is another approach instead of argparse using fire library. Currently, It is not considered due to the complexity of execution.

Reply:
The `typer` package might be a more supported and up-to-date solution to try:
https://github.com/tiangolo/typer

Perhaps beyond the scope of this MR, but one that we should consider using for this type of use case, generally speaking. It would probably be worth investigating how it handles some of these more complicated input params.